scripts/enable-datastore-viewer.sh

```diff
@@ -96,12 +100,17 @@ server {
     }
 }
 "
-	if [ -e /etc/nginx/sites-enabled/${app_id}.conf ]; then
-		cp /etc/nginx/sites-enabled/${app_id}.conf /tmp
-		echo "$pippo" >> /etc/nginx/sites-enabled/${app_id}.conf
-		echo "Datastore viewer enabled for $app_id at http://$(cat /etc/appscale/my_public_ip):$VIEWER_PORT. Allowed IP: $IP."
-		service nginx reload
-	else
-		echo "Cannot find configuration for ${app_id}."
-	fi
-done
+
+if [ -e /etc/nginx/sites-enabled/${APP_ID}.conf ]; then
+        echo "$config" > /etc/nginx/sites-enabled/${APP_ID}_datastore_viewer.conf
+        echo "Datastore viewer enabled for ${APP_ID} at http://$(cat /etc/appscale/my_public_ip):${VIEWER_PORT}. Allowed IP: $IP."
+        service nginx reload
+        if [ "$NO_OF_IP_LINES" != "2" ]; then
+                echo "Note: For a multi node deployment, you will need to forward the admin port from one of the AppServers on another node to the head node."
+                echo "You might need to type in the password for the app engine node."
+                echo "Run the SSH Tunnelling command: ssh -L ${LOCAL_PORT}:localhost:${port} ${APPENGINE_IP} -N"
+                echo "Modify the appscale/firewall.conf to open ports for the datastore viewer."
```

Comment:
The order of the instructions here is a bit weird.
I'd do the Modify... line first, then the Run... and the password at the end.

Are we gonna add the ssh key in the tunnel command afterall?